astroquery/alma/core.py

```diff
@@ -514,7 +605,8 @@ def _hack_bad_arraysize_vofix(self, text):
         return b"\n".join(newlines)
 
     def _login(self, username=None, store_password=False,
-               reenter_password=False, auth_urls=['asa.alma.cl',
+               reenter_password=False, auth_urls=['2020feb.asa-test.hq.eso.org',
```

Comment:
do we need the test server in the codebase?

Reply:
huh... now in this case... hm.  I've refactored.  Unclear.